modules/tests/test/framework/testoptions/test-options.go

```diff
@@ -79,47 +69,13 @@ func InitTestOptions(testOptions *TestOptions) error {
 	}
 	testOptions.Debug = strings.ToLower(debug) == "true"
 
-	testOptions.targetNamespaces = testOptions.resolveNamespaces()
-
 	testOptions.SkipCreateVMFromManifestTests = strings.ToLower(skipCreateVMFromManifestTests) == "true"
 	testOptions.SkipExecuteInVMTests = strings.ToLower(skipExecuteInVMTests) == "true"
 	testOptions.SkipGenerateSSHKeysTests = strings.ToLower(skipGenerateSSHKeysTests) == "true"
 
 	return nil
 }
 
-func (f *TestOptions) resolveNamespaces() map[constants.TargetNamespace]string {
-	var systemNS string
-	if f.DeployNamespace == "tekton-pipelines" {
-		systemNS = "default"
-	} else {
-		systemNS = "tekton-pipelines"
-	}
-
-	return map[constants.TargetNamespace]string{
-		constants.DeployTargetNS: f.DeployNamespace,
-		constants.TestTargetNS:   f.TestNamespace,
-		constants.SystemTargetNS: systemNS,
-		constants.EmptyTargetNS:  "",
-	}
-}
-
-func (f *TestOptions) ResolveNamespace(namespace constants.TargetNamespace, fallbackNamespace string) string {
-	if namespace == "" {
-		if fallbackNamespace != "" {
-			return fallbackNamespace
-		}
-	} else {
-		if namespace == constants.EmptyTargetNS {
-			return ""
-		}
-
-		ns := f.targetNamespaces[namespace]
-
-		if ns != "" {
-			return ns
-		}
-	}
-
-	return f.TestNamespace
+func (f *TestOptions) GetDefaultNamespace() string {
```

Comment:
```suggestion
func (f *TestOptions) GetDeployNamespace() string {
```

It's the namespace where everything was deployed, not necessarily the default one?

Reply:
fixed